components/ClusterAccordion.tsx

```diff
@@ -189,28 +190,44 @@ const ClusterAccordion = ({ clusters }: ClusterAccordionProps) => (
         <div className="col-span-6 grid grid-cols-subgrid text-center">
           <MetricBox className="col-start-2">
             <MetricLabel>
-              <MetricInfo label="open source">TODO: Popover details</MetricInfo>
+              <MetricInfo label="open source">
+                <TooltipContentHeader>open source</TooltipContentHeader>
+                Indicates whether the zkVM is open source or not
+              </MetricInfo>
             </MetricLabel>
           </MetricBox>
           <MetricBox className="col-start-3">
             <MetricLabel>
               <MetricInfo label="binary available">
-                TODO: Popover details
+                <TooltipContentHeader>binary available</TooltipContentHeader>
+                Indicates whether the necessary zkVM binary is available for
+                verification. This binary contains the compiled circuit and
+                verification keys essential for validating proofs
               </MetricInfo>
             </MetricLabel>
           </MetricBox>
           <MetricBox className="col-start-4">
             <MetricLabel>
               <MetricInfo label="avg cost">
-                Instruction set architecture
-                <br />
-                TODO: Popover details
+                <TooltipContentHeader>average cost</TooltipContentHeader>
+                Is calculated by multiplying the hourly cloud rate for
+                equivalent hardware by the number of instances used, then
+                multiplying by proof generation time (converted from
+                milliseconds to hours by dividing by 3,600,000)
               </MetricInfo>
             </MetricLabel>
           </MetricBox>
           <MetricBox className="col-start-5">
             <MetricLabel>
-              <MetricInfo label="avg time">TODO: Popover details</MetricInfo>
+              <MetricInfo label="avg time">
+                <TooltipContentHeader>average time</TooltipContentHeader>
+                <p>
+                  Is the average duration of computation time during the proof
+                  generation process, self reported by proving teams in
+                  milliseconds
+                </p>
+                <small>Time spent generating proof of execution</small>
```

Comment:
Would advise against usage of `small` here... 11.2px is too small

<img width="384" alt="image" src="https://github.com/user-attachments/assets/65514490-035f-4026-b95b-e0ec39f28f90" />

```suggestion
                <p>Time spent generating proof of execution</p>
```

Otherwise this all looks good